tests/unit/caffe_app/test_db.py

```diff
@@ -0,0 +1,52 @@
+import json
+import os
+import unittest
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.test import Client
+
+
+class SaveToDBTest(unittest.TestCase):
+
+    def setUp(self):
+        self.client = Client()
+
+    def test_save_json(self):
+        tests = open(os.path.join(settings.BASE_DIR, 'tests', 'unit', 'ide',
+                                  'caffe_export_test.json'), 'r')
+        net = json.load(tests)['net']
+        response = self.client.post(
+            reverse('saveDB'),
+            {'net': net, 'net_name': 'netname'})
+        response = json.loads(response.content)
+        self.assertEqual(response['result'], 'success')
+
+    def test_save_jsonstring(self):
```

Comment:
This is not required.

Reply:
Which part should be removed?

Reply:
The `test_save_jsonstring` function as both of them cover the same code.

Reply:
One is with the net passed in as a string while the other is passed in as an object. Not sure whether there is any difference in the underlying mechanism.

Reply:
Updated :D